encrypted-media-respec.html

```diff
@@ -1170,6 +1170,7 @@
                 </li>
                 <li><p>Load and initialize the <a def-id="keysystem"></a> implementation represented by this object's <var>cdm implementation</var> value if necessary.</p></li>
                 <li><p>Let <var>instance</var> be a new instance of the <a def-id="keysystem"></a> implementation represented by this object's <var>cdm implementation</var> value.</p></li>
+                <li><p>Initialize <var>instance</var> using <var>configuration</var>.</p></li>
```

Comment:
We might be explicit about the meaning of this. For example, that CDM features may be enabled/disabled/selected based on the contents of `<var>configuration</var>`.